### DISCUSSION TITLE
Is it possible to just give my app to review without releasing it to users?

### DISCUSSION THREAD

**Discussion by MikeMalec** [fastlane/fastlane#21070] [Q&A] [answered]
Is it possible to just give my app to review using fastlane deliver action(like in docs fastlane deliver --ipa "App.ipa" --submit_for_review) without releasing it to users and after some time start phased release with fastlane deliver submit_build?

**Comment by EmDee** [comment, marked as answer]
@MikeMalec Yes. The property you're looking for is `automatic_release` (https://docs.fastlane.tools/actions/deliver/), which is `false` by default (although it is not documented). Might be that it just uses the previous value in App Store Connect, so you might want to explicitly set this property to `false`.

**Reply by MikeMalec** [reply]
Great, thanks
I have one more question.
Let's say I submitted app for review but in the meantime I had to create a new app version and I submit it to review what will happen to previous one? Will it be cancelled automatically?

**Reply by EmDee** [reply]
@MikeMalec It depends. When the app has already been reviewed then I don't think you can automatically remove it, but have to manually cancel. If it's not in review yet, it _should_ automatically be cancelled if you pass `true` to the `deliver`'s `reject_if_possible` parameter. I say "should", because I personally have tried that.

**Reply by EmDee** [reply]
I can highly recommend reading the docs to each action you're using. The params section are usually quite extensively documented, and even if not, it helps to point you in the right direction.

**Reply by MikeMalec** [reply]
Thank you very much.
